sites/svelte.dev/src/routes/apps/destroy.js

```diff
@@ -1,11 +1,12 @@
 import * as gist from '$lib/db/gist';
 
-export async function post({ locals, body }) {
+export async function post({ locals, request }) {
```

Comment:
will this need to be `event` now?

Reply:
I think it's correct (though I still need to E2E test). The argument passed post is `event`, but I'm destructuring the request from it. https://kit.svelte.dev/docs#routing-endpoints-body-parsing

Reply:
ah, right. nevermind